sdk/python/feast/feature_store_config.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 The Feast Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional
+
+import yaml
+
+
+class Config:
+    """
+    Python representation of the FeatureStore config that the user can specify via a yaml file.
```

Comment:
Nit: If the user doesnt know what the yaml file is then this comment doesnt add value. How about `Configuration object that contains all possible configuration options for a FeatureStore`?